app/models/user.rb

```diff
@@ -110,6 +111,17 @@ def student?(course)
     course.users.role('student').include? self
   end
 
+  def campaign_organizer?
+    organized_campaigns.any?
+  end
+
+  def organized_campaigns
```

Comment:
Why not just use `.campaigns` in place of this? We don't have any other roles at this point, and if/when we do, we may still want to show all a user's campaigns.